main.py

```diff
@@ -1873,7 +1900,7 @@ def is_command(m):
         "get-record": record_get,
         "battletag": battletag,
         "set": set_command,
-        "whoami": whoami,
+        "me": whoami,
```

Comment:
Keep the old !whoami command as well for two releases. aka slow phase out old features as possible. just add:
`"whoami": whoami,`

And then we can remove when the surprise is less. Still can remove it from !help to help with the transition